client/components/Layout/FormEditScholarship.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Button, FormControl, Grid, Stack, TextField} from '@mui/material'
+
+function FormEditScholarship(){
+    return(
+        <Stack>
+            <FormControl component="form" sx={{ width: '100%' }}>
+                <TextField id="outlined-basic" label="Scholarship Name" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Provider Name" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Scholarship Quota" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Amount (Baht)" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Detail of Scholarship" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="GPAX" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Degree" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Program/Faculty" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Target Nation" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Field of interest" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Type of Scholarship" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+                <TextField id="outlined-basic" label="Application Deadline" variant="outlined" sx={{ padding: '0px 0px 20px 0px' }}/>
+            </FormControl>
```

Comment:
It would be beneficial to replace the standard TextField with the TextFieldComponent from @utils for validation purposes. Additionally, it would be useful to use props instead of copy-pasting the same TextField with only a different label value, as this would reduce duplication of TextFields and make the code more efficient.